README.md

```diff
@@ -4,18 +4,34 @@
 
 Simple interface to get the current time without binding to global system resources. PHP 5.6+
 
-## Usage
+## Clock Usage
 
 ```php
 function yourCode(Clock $clock) {
-	$time = $clock->now(); // No global access and easily testable
+    $time = $clock->now(); // Returns DateTimeImmutable. No global access and easily testable
```

Comment:
This comment is essential. When I looked at the Readme of the master branch, I wondered what the return type of `now` would be.